src/main/java/picard/sam/RevertSam.java

```diff
@@ -429,13 +429,15 @@ public void revertSamRecord(final SAMRecord rec) {
                         if (unpairedRecord == null) unpairedRecord = rec;
                         ++unpaired;
                     }
-                    if (rec.getFirstOfPairFlag()) {
-                        if (firstRecord == null) firstRecord = rec;
-                        ++firsts;
-                    }
-                    if (rec.getSecondOfPairFlag()) {
-                        if (secondRecord == null) secondRecord = rec;
-                        ++seconds;
+                    else {
```

Comment:
This code is not very good java... For the code you're updating, let's at least format the `else` correctly and put it on the previous line, e.g. `} else {`.

The `if`s should be formatted as:
```
if (firstRecord == null) {
    firstRecord = rec;
}
```